workflow-templates/release-go-task.yml

```diff
@@ -66,6 +68,9 @@ jobs:
         with:
           go-version-file: go.mod
 
+      - name: Setup QEMU
```

Comment:
Please evaluate using the alternative approach I proposed at https://github.com/arduino/serial-discovery/pull/94